images/router/clear-route-status.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# This allows for the clearing of route statuses, routers don't clear the routes status so some may be stale.
+# Upon deletion of the routes status active routers will immediatly update with a vaild status
+
+#clears status of all routers 
+function clear_status() {
+    local namespace="$1"
+    local route_name="$2"
+    MY_JSON_BLOB=$(oc get --raw http://localhost:8001/oapi/v1/namespaces/${namespace}/routes/${route_name}/)
+    MODIFIED_JSON=$(echo "$MY_JSON_BLOB" | jq 'del(.status.ingress)')
+    curl -s -X PUT http://localhost:8001/oapi/v1/namespaces/${namespace}/routes/${route_name}/status --data-binary "${MODIFIED_JSON}" -H "Content-Type: application/json" > /dev/null
+    echo "route status for route ${route_name} in namespace ${namespace} cleared"
+}
+
+#sets up clearing a status set by a specific router
+function clear_status_set_by() {
+    local router_name="$1"
+
+    NAMESPACES=$(oc get namespaces -o jsonpath='{range .items[*]}{.metadata.name} {end}')
```

Comment:
Does `-o jsonpath={.items[*].metadata.name}` work here? A little more `jsonpath`-ey and a little less `go-template`-ey